File: C++/titleToNumber.cpp
class Solution {
public:
    int titleToNumber(string s) {
       int result = 0;
       for (int i = 0; i < s.size(); i++) {
               result = result * 26 + (s[i] - 'A' + 1);
       }
       return result;
    }
};

File: Python/titleToNumber.py
class Solution:
    # @param A : string
    # @return an integer
    def titleToNumber(self, A):
        n = len(A)
        A = A[::-1]
        ret = 0
        for i in range(n):
            ret += (26**(i))*(ord(A[i])-ord('A')+1)
        return ret
